packages/gasket-plugin-data/lib/create.js

```diff
@@ -0,0 +1,24 @@
+/// <reference types="create-gasket-app" />
+
+const { name, version, devDependencies } = require('../package.json');
+
+/**
+ * @type {import('@gasket/core').HookHandler<'create'>}
+ */
+async function create(gasket, { pkg, files, gasketConfig }) {
```

Comment:
Does `gasketConfig.addPlugin` need to be called here to add the plugin to the app or is that not needed here?

Reply:
Yes good catch!